pkg/drivers/hyperv/powershell_windows.go

```diff
@@ -19,17 +17,8 @@ var (
 )
 
 func cmdOut(args ...string) (string, error) {
-	args = append([]string{"-NoProfile", "-NonInteractive"}, args...)
-	cmd := exec.Command("powershell.exe", args...)
-	log.Debugf("[executing ==>] : powershell.exe %v", strings.Join(args, " "))
-	var stdout bytes.Buffer
-	var stderr bytes.Buffer
-	cmd.Stdout = &stdout
-	cmd.Stderr = &stderr
-	err := cmd.Run()
-	log.Debugf("[stdout =====>] : %s", stdout.String())
-	log.Debugf("[stderr =====>] : %s", stderr.String())
-	return stdout.String(), err
```

Comment:
Sorry forgot to mention this earlier, we loose these debug statements, as `powershell.Execute` only logs when there's an error, and also the command getting executed will not be printed which is sometimes helpful to know in case of the commands being executed from the driver.

Reply:
Yes. Now logs are the same for the driver and crc code. I don't know which one is better. Should I log the command and stdout/stderr for all commands?

Reply:
Yes, i think log stdout/stderr for all commands, because this'll help us debug issues where there is no error but o/p is different then what we expect, but i can think of only the ipv6 scenario as of now where this helped..

Reply:
I added the command line log before each command. It is 100% like on Linux/macOS now.

``` 
logging.Debugf("Running '%s'", strings.Join(args, " "))
``` 

stdout/stderr are only log in case of failures.